ff_stdio.c

```diff
@@ -97,6 +97,7 @@ in a call to ff_truncate(). */
 
 	/* Only absolute paths are supported so define away the prvABSPath()
 	function. */
+	/*static*/ const char *prvABSPath( const char *pcPath );
```

Comment:
Please move the prototype up to the top of the file.  Can be above or below the prototype for prvFFErrorToErrno().  Thanks.